src/redux/books/books.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import Book from '../components/book/book-item';
-import NavBar from '../components/header/navigation';
-import Form from '../components/form/book-submit';
+import Book from '../../components/book/book-item';
+import NavBar from '../../components/header/navigation';
+import Form from '../../components/form/book-submit';
 
 const Books = () => (
   <div className="container">
```

Comment:
- [ ] It would be nice if this file is moved to components, let's redux hold all of redux related logics, and anything related to components constructions and displays stay in components folder for the sake of clean codes and readability